frontend-react/src/shared/Accordion/Accordion.module.scss

```diff
@@ -0,0 +1,19 @@
+@use "../../global-modules.scss" as *;
+
+/*
+ * Allows for Accordions with differing style expectations
+ */
+.usa-accordion--alternate {
```

Comment:
For CSS modules, the syntax we should use is something like:

```
.Accordian {

}
```

and in the component use it like `classNames={styles.Accordian}`

Reply:
and if we want an alternate using same syntax:

```
.Accordian {
  &Alternate {
  
  }
}
```

OR

```
.AccordianAlternate {
  
}
```

and can use, in either case, `classNames={styles.AccordianAlternate}`

Reply:
I don't agree. Not sure why we need to break away from USWDS syntax.

Reply:
@etanb this is a uswds component/component style. Why would we want to deviate from that?

Reply:
Because that's how CSS module syntax works: you create a CSS object essentially so you can use dot notation within the component. We might as well not use the `.module` syntax if we're going to deviate.

Reply:
Keeps the USWDS makes sense if we're overwriting pre-existing CSS, however if it's our own implementation of a **new** class, then it should be in the module syntax.

Reply:
Except we're making an enhancement of the existing USWDS component, not an entirely new original one so it is unnecessary to deviate from USWDS norms. Having to use two completely different naming systems, and especially in this case since your proposed names doesn't follow BEM, is confusing.

Reply:
CSS modules doesn't follow BEM in that `--` and `__` syntax.

Reply:
CSS modules has no opinion on name formatting outside of what it does automatically to make what's given unique, unless there's something I'm missing that you can reference? My Alert component enhancement uses BEM perfectly fine.

Reply:
@etanb considering that we are using this pattern throughout the app, can we table this and discuss at out eng sync? If we decide to change then we should update in the other areas as well.

Reply:
I've used it completely differently in the Navbar: https://github.com/CDCgov/prime-reportstream/blob/master/frontend-react/src/components/header/ReportStreamNavbar.module.scss

At the very least it should be:

```
.usa-accordian {
  &--alternate {}
}
```

But I also love using their variable names. We should choose something more custom to our app to denote that we're applying this unique style. We've never really discussed how we should do styling, and it's completely the wild west. I'd advocate for a top-level CSS module class that wraps the component, and then all the subsequent HTML elements can be styled directly with their classes (not using the `className={styles.cssClass}` syntax. Like:


// CSS

```
.Accordian {
  :global {
    .alternate-styling {
    }
  }
}
```

// Component

```
return (
<div className={styles.Accordian}>
  <div className={isAlternate && `alternate-styling`}>
  </div>
</div>
)
```